internal/cli/atlas_alertconfig.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func AtlasAlertConfigBuilder() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:        "alertConfigs",
+		Aliases:    []string{"alerts"},
```

Comment:
we can't use just `alerts` as we'll have `alerts` for other things
let's make the official command `alert-config` as the spec, feel free to add an alias for the camelCase version